app/assets/javascripts/tour/garden.xml.erb

```diff
@@ -0,0 +1,27 @@
+<krpano version="1.0.8.14">
+
+	<view hlookat="0" vlookat="0" maxpixelzoom="1.0" limitview="auto" />
+	<include url="<%= asset_path("tour/skin/defaultskin.xml") %>" />
+
+  <view fisheye="0" limitview="range" hlookatmin="-180" hlookatmax="180" vlookatmin="-90" vlookatmax="90" maxpixelzoom="1.0" fovmax="90" fov="90" hlookat="78" vlookat="10"/>
+  <test url="<%= Whitehall.public_asset_host %>/government/uploads/uploaded/number10/garden.tiles/preview.jpg" />

```

Comment:
Now this is `.erb` would it be worth extracting a bit more of this path into a helper to reduce the `<%= Whitehall.public_asset_host %>/government/uploads/uploaded/number10/` repetition?